elements/rh-spinner/rh-spinner.css

```diff
@@ -0,0 +1,81 @@
+:host {
+  display: inline-block;
+  aspect-ratio: 1 / 1;
+  text-align: center;
+  --rh-spinner--AnimationDuration: 1.2s;
+  --rh-spinner--StrokeWidth: 6px;
+  /*--rh-spinner--AnimationTimingFunction: ;*/
+}
+
+svg {
+  width: var(--rh-spinner--width, var(--rh-spinner--diameter, var(--rh-length-4xl, 64px)));
+  height: var(--rh-spinner--height, var(--rh-spinner--diameter, var(--rh-length-4xl, 64px)));
+  animation: rh-spinner-animation-rotate 
+      var(--rh-spinner--AnimationDuration)
+      ease
+      infinite;
+  overflow: hidden;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+circle {
+  width: 100%;
+  height: 100%;
+  stroke-width: var(--rh-spinner--StrokeWidth);
+  stroke-linecap: round;
+}
+
+circle.dash {
+  transform-origin: 50% 50%;
+  animation: rh-spinner-animation-dash 
+      var(--rh-spinner--AnimationDuration) 
+      ease-out
+      infinite;
+  stroke: var(--rh-color-blue-400, #0066cc);
+  stroke-dasharray: 283;
+  stroke-dashoffset: 280;
+  transform: rotate(-90deg);
+}
+
+circle.track {
+  stroke: var(--rh-color-black-300, #d2d2d2);
+  stroke-opacity: var(--rh-opacity-50, 50%);
+  animation-duration: 0;
+}
+
+
+:host([size="lg"]) svg {
+  --rh-spinner--diameter: var(--rh-spinner--lg--diameter, var(--rh-length-4xl, 64px));
+  --rh-spinner--StrokeWidth: 6px;
+}
```

Comment:
Any idea how to fix the discrepancy in the svg size and the circle size?

**svg**
![Screen Shot 2022-09-29 at 4 34 32 PM](https://user-images.githubusercontent.com/95588923/193137070-a8117948-5517-438e-9ab2-506a82da3c2c.png)

**circle**
![Screen Shot 2022-09-29 at 4 34 37 PM](https://user-images.githubusercontent.com/95588923/193137119-bab88527-39e1-4de3-84cc-bf0d12cfc575.png)

Reply:
Off the top, I think there are a few approaches you could take:
1. `calc()` the various diameters to take the width into account from the start
2. figure out a scaling factor and use `scale: ` on the svg element
3. perhaps max-width on the svg would do the trick